bofire/data_models/outlier_detection/outlier_detections.py

```diff
@@ -0,0 +1,53 @@
+import itertools
+from typing import List
+
+from bofire.data_models.base import BaseModel
+from bofire.data_models.domain.api import Inputs, Outputs
+from bofire.data_models.outlier_detection.outlier_detection import IterativeTrimming
+
+AnyOutlierDetector = IterativeTrimming
+
+
+class OutlierDetections(BaseModel):
+    """ "List of Outlier detectors.
+
+    Behaves similar to a outlier_detector."""
+
+    detectors: List[AnyOutlierDetector]
+
+    @property
+    def outputs(self) -> Outputs:
+        return Outputs(
+            features=list(
+                itertools.chain.from_iterable(
+                    [model.outputs.get() for model in self.detectors]  # type: ignore
+                )
+            )
+        )
+
```

Comment:
```suggestion
@validator(detectors)
def validate_detectors(cls,v,values):
    used_output_feature_keys = list(
            itertools.chain.from_iterable([detector.outputs.get_keys() for detector in v])
        )
        if len(set(used_output_feature_keys)) != len(used_output_feature_keys):
            raise ValueError("Output feature keys are not unique across surrogates.")
        return v

```

We need a validator here that ensures that the output detectors have different output features. And also a test for it.